modular_skyrat/modules/wirebrush_rust/code/game/objects/items/wirebrush.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/wirebrush
+	name = "wirebrush"
+	desc = "A tool that is used to scrub the rust thoroughly off walls. Not for hair or fur!"
+	icon = 'modular_skyrat/modules/wirebrush_rust/icons/obj/wirebrush.dmi'
+	icon_state = "wirebrush"
+	///variable that determines how long it takes to "clean" (read: replace(?)) rusted walls.
+	var/clean_speed = 30 //3 seconds
+
+/obj/item/wirebrush/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(istype(target, /turf/open/floor/plating/rust))
+		if(!do_after(user, clean_speed, target = target))
+			to_chat(user, span_warning("You stop cleaning, leaving the wall still rusty!"))
```

Comment:
```suggestion
			to_chat(user, span_warning("You stop cleaning, leaving the plating still rusty!"))
```

Reply:
alternatively, you could create the rustedTurf var before the do_after, and then just use [rustedTurf]

Reply:
solved this issue @tf-4